modules/vba_documents/app/sidekiq/vba_documents/report_monthly_submissions.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
+require 'sidekiq/monitored_worker'
```

Comment:
Without this line, the specs wouldn't run successfully. This was a prior oversight.